cms/utils/reversion_hacks.py

```diff
@@ -1,6 +1,26 @@
 # -*- coding: utf-8 -*-
-import reversion
-from reversion.revisions import RegistrationError, VersionAdapter
+
```

Comment:
Well, I could easily change this in a simple version checking.
Having a simple try / except will make easier to remove the legacy imports in 3.3 (as we can use reversion 1.10 for all the supported Django versions) and a single section makes the code more readable. I'll happily add a comment to clarify the reason of the code style